### PR TITLE
dhcp: SV Test to validate dhcp option 60 is being logged

### PR DESCRIPTION
**Pull request by yatink**
## Feature 4587

This PR adds a test to validate that dhcp option 60 is being logged in dhcp extended logging mode. 

Redmine ticket: [4587](https://redmine.openinfosecfoundation.org/issues/4587)

### CODE REVIEW

File: tests/dhcp-eve-extended-option-60/test.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# *** Add configuration here ***
```
Comment (inashivb) [inline tests/dhcp-eve-extended-option-60/test.yaml:1]: Could you please remove this line and add
```
requires:
  min-version: 7
```
to make sure this only runs on 7 as this feature has just been added?

**Review by inashivb — CHANGES_REQUESTED** [review]
Hi @yatink ! Nice job making changes. :)
A few small changes to be made:
1. Please make sure there is just one commit in this PR for this issue. Squash the fix with the main commit.
2. Drop `SV Test to` from the commit message

**Comment by yatink** [comment]
@inashivb, can I squash the commits and force push to this branch itself or should I open a new PR ?

**Comment by inashivb** [comment]
> Force pushing for now...happy to put my changes into a new branch and open a new PR if that's preferable.

That's always preferable. ;)

**Comment by yatink** [comment]
> > Force pushing for now...happy to put my changes into a new branch and open a new PR if that's preferable.
> 
> That's always preferable. ;)

Got it ! I'll create a new one.

**Comment by yatink** [comment]
Continued in https://github.com/OISF/suricata-verify/pull/1359

**Pull request closed without merging**